style.css

```diff
@@ -666,3 +664,421 @@ textarea {
+  }
+
+  .right-block {
+    grid-row: 1 / 2;
+    grid-column: 2 / 3;
+    width: 100%;
+    height: 70%;
+    display: inline-block;
+  }
+
+  .block-h2 {
+    width: 100%;
+    height: fit-content;
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  .block-p {
+    width: 80%;
+    height: fit-content;
+  }
+
+  .block-nav {
+    width: 100%;
+  }
+
+  .block-nav > ul {
+    padding: 0;
+  }
+
+  .nav-list {
+    width: 70%;
+    height: 70%;
+  }
+
+  .right-btn {
+    width: 100%;
+    height: 60%;
+  }
+
+  .frame43 {
+    width: 100%;
+    height: auto;
+    grid-row: 2 / 3;
+    grid-column: 1 / 3;
+    object-fit: cover;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+  }
+
+  .card-work {
+    width: 100%;
+    height: 100%;
+  }
+
+  .group {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 0;
+  }
+
+  .group-header {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-wrap: nowrap;
+    font-size: 2vw;
+    line-height: 5vh;
+  }
+
+  .group-para {
+    width: 80%;
+    height: 100%;
+    font-size: 1.5vw;
+    line-height: 4vh;
+  }
+
+  .group-ul {
+    width: 100%;
+    height: 10vh;
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .group-li {
+    width: 100%;
+    height: 50%;
+    font-size: 1.3vw;
+    line-height: 5vh;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .bootstrap {
+    flex-grow: 0.5;
+  }
+
+  .ruby-html {
+    flex-grow: 0.3;
+  }
+
+  .bg-printing {
+    background-image: url("images/bg-one.svg");
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .hide {
+    visibility: hidden;
+  }
+
+  .bg-availability {
+    background-image: url("images/bg-two-five.svg");
+  }
+
+  .background {
+    background-image: url("images/bg-three-six.svg");
+  }
+
+  .bg-print {
+    background-image: url("images/bg-four.svg");
+  }
+
+  .btn-hide {
+    visibility: hidden;
+  }
+
+  .card-group {
+    display: flex;
+    flex-direction: column;
+    gap: 0;
+    padding-top: 110px;
+    padding-left: 10px;
+  }
+
+  .about-me {
+    margin-top: 0;
+    width: 100%;
+    height: fit-content;
+    padding: 7vh 0 0 7vw;
+    background-image: url("images/Frame.svg");
+    background-position: 100% 12%;
+    background-size: 35% 50%;
+  }
+
+  .about-me > h2 {
+    margin-bottom: 1vh;
+  }
+
+  .about-me > p {
+    width: 45%;
+    margin-top: 0;
+    margin-bottom: 1vh;
+  }
+
+  .about-me > button {
+    margin-top: 1vh;
+  }
+
+  #lang {
+    width: 90%;
+    margin-top: 10vh;
+    margin-bottom: 10vh;
+    display: flex;
+    justify-content: flex-start;
+    padding-left: 0;
+    border-top: 1px solid #fff;
+  }
+
+  #lang > ul {
+    width: 100%;
+    padding-left: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    gap: 10%;
+    margin: 0 7vw;
+  }
```

Comment:
[OPTIONAL] Please try to fix the layout of your language section as it is currently not centered your design 👇 
![image](https://user-images.githubusercontent.com/97521557/201492182-cd1d9df8-3639-4a8d-b6d7-ef41dcbf24be.png)
Figam design 👇 
![image](https://user-images.githubusercontent.com/97521557/201492194-09822a1f-5dce-4b51-8b8f-958c848db4ee.png)

I think you can remove the` gap` property from #lang > ul and  use` justify-content: space-around`